deployment/checkCmd.py

```diff
@@ -0,0 +1,17 @@
+# TODO it's a dummy impl now
+class CheckCmd():
```

Comment:
Can you create a new module named ```cmd``` to put the command file?

Reply:
Hi @ydye ,

I don't want to increase the directory hierarchy for now, because now is clear enough.